src/components/room-item.js

```diff
@@ -1,17 +1,31 @@
-import React from 'react';
+import React, {Component} from 'react';
 import styles from './room-item.module.scss';
 
-class RoomItem extends React.Component {
+class RoomItem extends Component {
+
+  handleClick = event => {
+    event.preventDefault();
+    const {id, isSelected, selectRoom} = this.props;
+
+    if(!isSelected){
```

Comment:
use space between `if`  and opening bracket `(` and between closing bracket `)` and `{`